src/pages/auth/Login.tsx

```diff
@@ -38,17 +38,40 @@ export default function Login() {
   const isLoginDisabled = !(username && password);
   const navigate = useNavigate();
   const [modalContent, setModalContent] = useState({
+    type : "username",
     title: "",
     message: "",
     example: "",
   });
   const { isOpen, onOpen, onClose } = useDisclosure();
 
-  const openModal = (title: string, message: string, example: string) => {
-    setModalContent({ title, message, example });
+  const { t } = useTranslation();
+
+  const state = import.meta.env.VITE_APP_STATE;
+
+  const getModalExample = (type: "username" | "password") => {
+    if (state === "KA") {
+      return type === "username"
+        ? t("LOGIN_USERNAME_EXAMPLE_EXPLANATION_KA")
+        : t("LOGIN_PASSWORD_EXAMPLE_EXPLANATION_KA");
+    } else {
+      return type === "username"
+        ? t("LOGIN_USERNAME_EXAMPLE_EXPLANATION_OD")
+        : t("LOGIN_PASSWORD_EXAMPLE_EXPLANATION_OD");
+    }
+  };
```

Comment:
What is state, and what is the difference between the state having 'KA' and being empty or undefined?